ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java

```diff
@@ -104,6 +102,11 @@ public void verifyThereAreWarnings(int num) {
         assertEquals("Wrong number of warnings", num, actual);
     }
 
+    @Then("^verify API spec warnings contain \"([^\"]*)\"$")
+    public void verifyWarningsContain(String warning) {
+        assertTrue(reviewApiProviderActions.getErrors().stream().anyMatch(s -> s.contains(warning)));
```

Comment:
`assertTrue` will give you a very unhlepful failure message along the lines "expected [true], found [false]".
There's actually an `anyMatch` assertion which should work as a drop-in replacement for your code, except the message will be much nicer.